mlrun/projects/pipelines.py

```diff
@@ -940,6 +943,11 @@ def create_pipeline(project, pipeline, functions, secrets=None, handler=None):
         handler = "pipeline"
     if not handler or not hasattr(mod, handler):
         raise ValueError(f"pipeline function ({handler or 'pipeline'}) not found")
+        raise ValueError(
```

Comment:
Will this really be raised after the exception in the line above is raised?

Reply:
oopsi